UnitTests/BraintreeShopperInsightsTests/BTShopperInsightsClient_Tests.swift

```diff
@@ -227,4 +233,38 @@ class BTShopperInsightsClient_Tests: XCTestCase {
         sut.sendVenmoSelectedEvent()
         XCTAssertEqual(mockAPIClient.postedAnalyticsEvents.first, "shopper-insights:venmo-selected")
     }
+
+    // MARK: - App Installed Methods
+
+    func testIsPayPalAppInstalled_whenPayPalAppNotInstalled_returnsFalse() {
+        let fakeApplication = FakeApplication()
+        payPalClient.application = fakeApplication
+        fakeApplication.cannedCanOpenURL = false
+
+        XCTAssertFalse(payPalClient.application.isPayPalAppInstalled())
```

Comment:
I think these should all be asserting on `sut. isPayPalAppInstalled()` or `sut.isVenmoAppInstalled()`

Reply:
Thanks, made some changes:

```
    func testIsPayPalAppInstalled_whenPayPalAppIsInstalled_returnsTrue() {
        let fakeApplication = FakeApplication()
        fakeApplication.cannedCanOpenURL = true
        fakeApplication.canOpenURLWhitelist.append(URL(string: "paypal-app-switch-checkout://x-callback-url/path")!)
        sut.application = fakeApplication

        XCTAssertTrue(sut.isPayPalAppInstalled())
    }
```

+ similar change for Venmo